foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -815,6 +830,11 @@ inline void Server<ServerConfiguration>::sendMessage(ConnHandle clientHandle, Ch
 
   const auto bufferSizeinBytes = con->get_buffered_amount();
   if (bufferSizeinBytes >= _send_buffer_limit_bytes) {
+    FOXGLOVE_DEBOUNCE(
+      [this](websocketpp::log::level level, const std::string& msg) {
+        _server.get_elog().write(level, msg);
+      },
```

Comment:
why not just `[&] { _server.get_elog().write(WARNING, "..."); }` ?

Reply:
That's what I had initially, this gave however errors for melodic:
```
ISO C++11 requires at least one argument for the ...'' in a variadic macro
```

Reply:
I suppose you could just not allow variadic args.

Reply:
Will remove it